src/Files.App/App.xaml.cs

```diff
@@ -73,6 +74,7 @@ public partial class App : Application
 
 		public static string AppVersion = $"{Package.Current.Id.Version.Major}.{Package.Current.Id.Version.Minor}.{Package.Current.Id.Version.Build}.{Package.Current.Id.Version.Revision}";
 		public static string LogoPath;
+		public static AppEnvironment AppEnv;
```

Comment:
This doesn't really belong in App.xaml.cs but we can patch it up in another PR.

Reply:
Ideally we can build a service that has values for `Environment`, `LogoPath`, `Version` etc.

Reply:
Ok

Reply:
> This doesn't really belong in App.xaml.cs but we can patch it up in another PR.

Should I do it after this is merged?

Reply:
Yes, it's out of scope for this PR.